### ISSUE TITLE
default hud bugs

### ISSUE THREAD

**Issue by Hulkfog** [Kamikaze94/WolfHUD#31]
When stealth is broken the detection risk is still showed
The accuracy panel gets hidden by special equipment  such as planks

**Comment by Kamikaze94** [comment]
Okay.
Risk Indicator:
That whisper mode change for default HUD was a crappy workaround I did back then. It required you to take health damage (maybe even in stealth and then in loud) to refresh. Replaced that now with a simple listener call of GameInfoManager.
For the Accuracy Panel: 
I moved the Special Equipment above the Accuracy text now, so both are visible.
If you pick up a bag, your special equipment gats behind that semi-transparent box, but I guess thats not too big of an issue...

Feel free to test it out, as I just gave it a short run, to make sure it doesn't instant crash... ^^

OFFTOPIC:
While testing with the DownCounter bug, I found out, the Damage Popup shows for the bot you are spectating while in custody...
Seems stange, need to evaluate if I should keep it or prevent that... :stuck_out_tongue_winking_eye:
